### PR TITLE
Changed SendRequest to output headers

### PR DESCRIPTION
**Pull request by brett-wright**

### CODE REVIEW

File: client/client.go [hunk 1]
```diff
@@ -50,41 +52,51 @@ func (c *Client) SendRequest(method string, path string, payload interface{}, st
...
 	}
 
-	return strbody, nil
+	return strbody, string(headers), nil
+}
+
+// GetID gets the resource id from locaation resposne header
+func GetID(body string) (id string) {
+	var jsonData models.ResponseHeaders
+	json.Unmarshal([]byte(body), &jsonData)
```
Comment (lassegaardsholt) [inline client/client.go:93]: Remember to check for errors

**Review by lassegaardsholt — CHANGES_REQUESTED** [review]

File: client/client.go (hunk 1 above)
Comment (brett-wright) [reply client/client.go:93]: lol I knew you were going to ask about this. But it was getting me the below error
`cannot use err (variable of type error) as string value in return statement`

Comment (brett-wright) [reply client/client.go:93]: Dont worry I worked it out.
`func GetID(body string) (id string, err error) {
	var jsonData models.ResponseHeaders
	err = json.Unmarshal([]byte(body), &jsonData)
	if err != nil {
		return "", err
	}

	localation := jsonData.Location[0]
	localation = strings.Replace(localation, "/api", "", -1)

	// removes /v2.0 from string if using api version 2
	localation = strings.Replace(localation, "/v2.0", "", -1)

	return localation, nil
}
`

**Review by peterbrondum — APPROVED** [review]
LGMT

**Comment by lassegaardsholt** [comment]
Make sure it can compile, and I think it looks fine 👍

**Review by lassegaardsholt — APPROVED** [review]

**Pull request merged**